UniWatch/Controllers/StudentController.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using UniWatch.DataAccess;
+using UniWatch.Models;
+using UniWatch.ViewModels;
+
+namespace UniWatch.Controllers
+{
+    public class StudentController : Controller
+    {
+        private IDataAccess _dataAccess;
+
+        public StudentController() : this(new DataAccess.DataAccess())
+        { }
+
+        public StudentController(IDataAccess dataAccess)
+        {
+            _dataAccess = dataAccess;
+        }
+
+        // GET: /Student/Index?classId=1
+        /// <summary>
+        /// Display a list of students in a class
+        /// </summary>
+        /// <param name="classId">The id of the class</param>
+        [HttpGet]
+        public ActionResult Index(int classId)
+        {
+            var @class = _dataAccess.ClassManager.GetById(classId);
+            if (@class == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
```

Comment:
Add a `string` inside the `HttpStatusCodeResult` explaining why the request is bad.

**i.e.** _No class with id [classId]_